projects/expat/Dockerfile

```diff
@@ -15,7 +16,13 @@
 ################################################################################
 
 FROM gcr.io/oss-fuzz-base/base-builder
-RUN apt-get update && apt-get install -y cmake docbook2x make
+RUN apt-get update && apt-get install -y \
+            cmake \
```

Comment:
Hm, how did we get proto before?

Reply:
@vitalybuka not at all, Protobuf is a new dependency upstream only since https://github.com/libexpat/libexpat/pull/950 .